src/controllers/task.js

```diff
@@ -19,9 +21,40 @@ const createTask = async task => {
   return response;
 };
 
-const readAllTasks = () => {};
+const readAllTasks = async tasks => {
+  //const allTasks = Task.find({});
```

Comment:
If this line is no longer needed, just remove it. Getting in the habit of leaving a bunch of commented out code will make it harder to follow what you were doing later. Comments should not be used to "save" old code. They should be used to document chunks of code (like functions, or classes), or to explain any oddities in the code ("This for loop is weird because...")